src/corehost/corehost.cpp

```diff
@@ -267,6 +267,14 @@ bool get_latest_fxr(pal::string_t fxr_root, pal::string_t* out_fxr_path)
 
 #if defined(CURHOST_LIB)
 
+namespace
+{
+    void libhost_default_error_writer(const pal::char_t*)
```

Comment:
I would call this `libhost_null_error_writer` - default is really the stderr one.